designer/client/api/toggleApi.ts

```diff
@@ -0,0 +1,11 @@
+export class ToggleApi {
+  async fetchToggles() {
+    try {
+      const response = await window.fetch("/feature-toggles");
+      return response.json();
+    } catch (e) {
+      console.error(e);
+      return [];
```

Comment:
are these matching return types? it looks like response is an object

Reply:
What gives you that impression? I thought `/feature-toggles` would be enough to show its a list

Reply:
ohhh I see. ` result["featureEditPageDuplicateButton"] ` this gave me the impression, it would work for object as well. maybe it would be best as an object so you can destructure it?

Reply:
@endamccormack Jen was right on this... `getAllResolved` returns an object, so the API at `/feature-toggles` is returning an object.

Reply:
Oh no, I know she was right, when I wrote that message I had forgotten that I had already changed it to an object as it was originally an array.

I mean if we're doing code reviews after the fact, I have some notes 😉❤️

Reply:
☕️... I'll leave you boys to it